statsrunner/loop.py

```diff
@@ -10,10 +10,22 @@
 import statsrunner.aggregate
 from statsrunner.common import decimal_default
 
+
 def call_stats(this_stats, args):
+    """Create dictionary of enabled stats for this_stats object.
+
+    Args:
+      this_stats: stats_module class object that specifies calculations for
+      relevant input, processed by internal methods of process_file().
```

Comment:
It can be ignored, yep...

Because of this, the line length limit in `iati.core` is set at 255